DashKit/Mocks/MockPodCommManager.swift

```diff
@@ -0,0 +1,234 @@
+//
+//  MockPodCommManager.swift
+//  DashKitTests
```

Comment:
Was this file moved largely unchanged?   (P.S. super-nit, but this comment about it being in `DashKitTests` may need to change to its new location.)